pyrato/parameters.py

```diff
@@ -102,3 +103,113 @@ def reverberation_time_linear_regression(
         return reverberation_times, intercepts
     else:
         return reverberation_times
+
+
+
+
+
+
+def clarity(energy_decay_curve, early_time_limit=80):
+    r"""
+    Calculate the clarity from the energy decay curve (EDC) of
+    a room impulse response.
```

Comment:
```suggestion
    Calculate the clarity from the energy decay curve (EDC).
```
This first sentence get's rendered as the short summary and therefore should be brief